app/models/post.rb

```diff
@@ -62,4 +62,16 @@ class Post < ApplicationRecord
       transitions from: [:published, :active, :inactive], to: :archived
     end
   end
+
+  def self.search(search)
+    if search
+      attributes = ["title", "description", "content"]
+      queries = attributes.map { |attr| "(#{attr} LIKE '%#{search}%')" }
+      built_query = queries.join(" OR ")
+      where(built_query)
```

Comment:
seems vulnerable to SQL injection :(
https://stackoverflow.com/questions/25416667/rails-4-prevent-sql-injection-using-like-operator-in-sql-query